addons/account/models/account_move.py

```diff
@@ -1921,6 +1921,7 @@ def create_tax_cash_basis_entry(self, percentage_before_rec):
         if newly_created_move:
             self._set_tax_cash_basis_entry_date(move_date, newly_created_move)
             # post move
+            newly_created_move.update_lines_tax_exigibility()
```

Comment:
You don't need this line because the cash basis journal items are directly created with `tax_exigible=True`